plugin/src/main/scala/org/flywaydb/play/PlayInitializer.scala

```diff
@@ -113,6 +113,9 @@ class PlayInitializer @Inject() (implicit app: Application, webCommands: WebComm
     webCommands.addHandler(flywayWebCommand)
 
     for (dbName <- allDatabaseNames) {
+      if (flywayConfigurations(dbName).cleanOnStart) {
```

Comment:
> the difference between "test" and "production" is the way the application is started. 

flyway-play is a play module and have to obey play's way rather than your use cases.

Please call Java API `clean` on the `onStart` hook of your application like following

```
val flyway = new Flyway
flyway.clean()
```